components/director/Dockerfile

```diff
@@ -7,17 +7,11 @@ WORKDIR ${BASE_APP_DIR}
 # Copy files
 #
 
-COPY ./internal/ ${BASE_APP_DIR}/internal/
-COPY ./pkg/ ${BASE_APP_DIR}/pkg/
-COPY ./vendor/ ${BASE_APP_DIR}/vendor/
-COPY ./cmd/ ${BASE_APP_DIR}/cmd/
-COPY ./hack/ ${BASE_APP_DIR}/hack/
-COPY ./licenses/ ${BASE_APP_DIR}/licenses/
-
-#
+COPY . .
```

Comment:
Then we copy not only needed files, but also scripts, makefile, dockerfile and gitignore. I would keep it as it is, as the list here won't change anywhere soon.

Reply:
but we copy to builder image, not to the destination image

Reply:
Fair enough, you can leave it as it is

Reply:
Yup,  but it will be slightly slower, while copying local binary and other stuff... but anyway, it's not a blocker of this PR